eth/consensus/clique/clique.py

```diff
@@ -85,22 +87,23 @@ def __init__(self, base_db: AtomicDatabaseAPI, epoch_length: int = EPOCH_LENGTH)
             self._epoch_length,
         )
 
-    @to_tuple
-    def amend_vm_configuration(self, config: VMConfiguration) -> Iterable[VMFork]:
```

Comment:
I agree but I don't have a good answer. With the old approach you always end up having different classes then those defined in `cls.vm_configuration`. We have tests that assert things like `assert chain.vm_configuration == RopstenChain.vm_configuration` that break because of that. Is there an alternative (apart from formalizing every method that needs to be replaced in an interface and delegating to it)